blog/20140130-mate-desktop-live-cd.md

```diff
@@ -15,7 +15,7 @@ The MATE team have made a Live CD that boots into a full MATE desktop.
 This Live CD was created so that potential new users can evaluate the MATE 
 desktop in a non-destructive fashion. The image can be burned to a DVD, 
 mounted as an ISO file, or be directly written to a USB stick using a utility 
-like `dd`. [Linux Luddites](http://linuxluddites.com/) beware, `unetbootin` is
+like `dd`. [Linux Luddites](https://linuxluddites.com/) beware, `unetbootin` is
```

Comment:
https://linuxluddites.com/ seems to be a dead page, it shows "hello world" page

in the past, it redirects to https://luddites.latenightlinux.com

See https://web.archive.org/web/20191015031023/http://linuxluddites.com/

![2019-12-08_16-15](https://user-images.githubusercontent.com/7734191/70391486-1e48ba80-19d6-11ea-9091-be841e5643aa.png)

I suggest to use https://luddites.latenightlinux.com instead

Reply:
switched to https://luddites.latenightlinux.com